music/src/core/compat/global.ts

```diff
@@ -31,3 +37,17 @@ export interface Performance {
 export const fetch: typeof window.fetch = isNode ? require('node-fetch') : window.fetch.bind(window);
 export const performance: Performance = isNode ? require('./performance_node') : window.performance;
 export const navigator = isNode ? require('./navigator_node') : window.navigator;
+
+export function isSafari() {
+  if (isNode) {
+    throw new Error('Cannot check if Safari in Node.js');
+  }
+  return /^((?!chrome|android).)*safari/i.test(navigator.userAgent);
+}
+
+export function getOfflineAudioContext(sampleRate: number): OfflineAudioContext {
+  if (isNode) {
+    throw new Error('Cannot use offline audio context in Node.js');
+  }
+  return new window.OfflineAudioContext(1, sampleRate, sampleRate);
```

Comment:
I think this needs to be the if Safari? webkitOfflineAudioContext : offlineAudioContext, since you could be realistically developing and testing on Safari

Reply:
Yeah, I thin that's right

Reply:
i dont think this change went through?

Reply:
Yup, you're right